bench/bench.pony

```diff
@@ -14,7 +14,7 @@ actor Main is BenchmarkList
     bench(_MultipartFileUploadBenchmark)
     bench(_ChunkedRequestBenchmark)
 
-actor _TestSession is Session
+actor _TestSession is (Session & HTTP11RequestHandler)
```

Comment:
This was something that we forgot to include back in 0.47.0

Reply:
Can you remove this from this PR and put into another PR?